src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: FC<TodoInfoProps> = ({ todo }) => {
+  // const user = usersFromServer.find(({ id }) => id === todo.userId);
+
+  // if (!user) {
+  //   return null;
+  // }
+
+  return (
+    <article
+      key={todo.id}
+      data-id={todo.id}
+      className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
Ensure that the `todo.user` property is correctly passed from the parent component. If there's a chance that `todo.user` could be undefined, consider adding a check or fallback to handle such cases gracefully.